### PR TITLE
[onert] Write runtime/api.md

### PR DESCRIPTION
**Pull request by wateret**
This commit fills the content of `runtime/api.md`.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

File: docs/runtime/api.md [hunk 1]
```diff
@@ -1 +1,35 @@
...
+
+Here is a figure of runtime layered architecture.
+
+![Layered Architecture](api-layered-arch.png)
+
+There are three parts - Frontend, Core and Backend. Core works with Frontend and Backend API. Frontend gets user inputs(neural networks models) and Backend does the actual computation.
+
+## Frontend API
+
+Frontend API is about from creation/loading the model and
```
Comment (hyunsik-yoon) [inline docs/runtime/api.md:13]:
```suggestion
Frontend API is for creation/loading the model and
```

File: docs/runtime/api.md [hunk 2]
```diff
@@ -1 +1,35 @@
...
+
+For usage, refer to [Howto : NN API](../howto/how-to-use-nnapi-binding.md).
+
+### NNFW API
+
+NNFW API is ONE's own API. It supports loading models from NN Packages. As it is our own API, It can do most of functionalities that One Runtime offers. Representatively, it provides functions for execution with multiple backends.
+
+For usage, refer to [Howto : NNFW API](../howto/how-to-use-nnfw-api.md).
+
+## Backend API
```
Comment (lemmaa) [inline docs/runtime/api.md:29]: @wateret , Where can I find the `Backend API` in the diagram?

Comment (wateret) [reply docs/runtime/api.md:29]: It is "Backend Interface" in the figure. I will change it in the figure.

Comment (hyunsik-yoon) [reply docs/runtime/api.md:29]: I tried to render png and md but it seems that github cannot show `api-layered-arch.png`. Could you check?

**Pull request merged**